scripts/cluster_network.py

```diff
@@ -362,7 +362,8 @@ def consense(x):
             )
             return v
         potential_mode = consense(pd.Series([snakemake.config['renewable'][tech]['potential']
-                                             for tech in renewable_carriers]))
+                                             if tech.split('-', 2)[0] in snakemake.config['renewable']])
+                                  
```

Comment:
What's this change about? Maybe rather another PR because not related to OPSD ppls?

Reply:
That's the PR #191 you mentioned before and what I meant in my comment above.

Reply:
So that would have belonged to #191? Hmm, you could add a commit that reverts this change on this branch `eb5194:OPSD_renewable_ppls`, make a new branch from the current `master` and make a PR with only this change. Then we have separated the changes.

Reply:
PR into this branch?

Reply:
No, PR into `master`.

Reply:
and then remove/close this PR?

Reply:
No, we just want to split this PR into two (one for OPSD stuff, the other for fixing the clustering). You keep this PR for OPSD, the other only has the changes in `cluster_network.py`.

Reply:
Ok, then done. The clustering changes are already in the master branch, so no need for a PR, this was exactly #191, but for some reason it appeared in this RP as well. After my latest commit, cluster_network.py is the same as in the master branch

Reply:
Ok, perfect!